SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaDatabase.h

```diff
@@ -56,6 +56,16 @@ class SPATIALGDK_API USchemaDatabase : public UDataAsset
 		return SpatialConstants::INVALID_COMPONENT_ID;
 	}
 
+	uint32 GetComponentIdForClass(const UClass& Class) const
```

Comment:
I'm opposed to modifying SchemaDatabase.h since it should just have data definitions and not much logic. I think this would be a great function to put into the ClassInfoManager.h

Reply:
Agree, will do in a separate change.

Reply:
https://improbableio.atlassian.net/browse/UNR-1626